js/core.js

```diff
@@ -1351,6 +1351,65 @@ var ERMrest = (function (module) {
         };
 
         /**
+         * Formats the presentation value corresponding to this column definition.
+         * @param {String} data The 'formatted' data value.
+         * @param {Object} options The key value pair of possible options with all formatted values in '.values' key
+         * @returns {Object} A key value pair containing value and isHTML that detemrines the presenation.
+         */
+        this.formatPresentation = function(data, options) {
+            
+            var utils = module._formatUtils, keyValues = options.keyValues, columns = options.columns;
+
+            /*
+             * TODO: Add code to handle `pre_format` in the annotation
+             */
+
+            var isMarkdownPattern = false, isMarkdownType = false;
+
+            if (this.annotations.contains(module._annotations.COLUMN_DISPLAY) && this.annotations.get(module._annotations.COLUMN_DISPLAY).contains("markdown_pattern")) {
+                isMarkdownPattern = true;
```

Comment:
Looking at the annotation document, the payload of column display annotation should be 
`{context : { "markdown_pattern": ... }}`. The code assumes the format `{"markdown_pattern": ...}`